ospdo adoption ns overhaul (#873)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/con_adoption-prerequisites.adoc

** link:{defaultOCPURL}/nodes/overview-of-nodes[Overview of nodes]
** link:{defaultOCPURL}/nodes/index#nodes-scheduler-node-selectors-about_nodes-scheduler-node-selectors[About node selectors]
** link:{defaultOCPURL}/machine_configuration/index[Machine configuration overview]
* Make sure to set the correct {rhos_acro} project namespace to execute commands in.
ifeval::["{build_variant}" == "ospdo"]
[NOTE]
In Director Operator adoption the adopted namespace would be "openstack", the adopting namespace would be different i.e. "rhoso".
endif::[]
[source, shell]
----

> $ oc project $NAMESPACE

Reviewer: I would just use the literal example `oc project openstack` that majority of customers will likely want to use, instead of using `$NAMESPACE`.

Author: I think @bogdando asked for this to be parameterized no ? 
and for the ospdo version that would be the wrong namespace to use , so how to get around that ? an if condition?

Reviewer: Yes either an if condition or just the `[NOTE]` that you already included?

Author: ok

Reviewer 2: done

** link:{defaultOCPURL}/nodes/overview-of-nodes[Overview of nodes]
** link:{defaultOCPURL}/nodes/index#nodes-scheduler-node-selectors-about_nodes-scheduler-node-selectors[About node selectors]
** link:{defaultOCPURL}/machine_configuration/index[Machine configuration overview]

> * Make sure to set the correct {rhos_acro} project namespace to execute commands in.
> ifeval::["{build_variant}" == "ospdo"]
> In Director Operator adoption the adopted namespace would be "openstack", the adopting namespace would be different i.e. "rhoso".

Reviewer 3: ```suggestion
* Make sure to set the correct {rhos_acro} project namespace in which to run commands.
ifeval::["{build_variant}" == "ospdo"]
In director Operator adoption, the adopted namespace is `openstack`. The adopting namespace is different, for example, `rhoso`.
```
What is an "adopted" namespace vs. an "adopting" namespace?

Reviewer 2: Adopted is where OSP17 director is deployed(source and adopting is RHOSO 18 namespace(destination). @klgill How to rephrase it to make it understandable?

> In director Operator adoption, the adopted namespace is `openstack`. The adopting namespace is different, for example, `rhoso`.

Reviewer 3: ```suggestion
In director Operator adoption, the source {rhos_prev_long} {rhos_prev_ver} namespace is `openstack`, and the destination {rhos_acro} {rhos_curr_ver} namespace is different, for example, `rhoso`.
```
This sentence renders to the following in the doc:
In director Operator adoption, the source Red Hat OpenStack Platform 17.1 namespace is `openstack`, and the destination RHOSO 18.0 namespace is different, for example, `rhoso`.

Does that make sense? ^

Reviewer 2: First part of sentence is correct, but in the second part we should highlight the fact that the destination namespace must be different, for example rhoso. The namespace must be different in order to adopt succesfullly.

Reviewer 3: @ciecierski I edited the suggestion above. WDYT?

Reviewer 2: I added your suggested text.
